packages/autocomplete-preset-algolia/package.json

```diff
@@ -31,7 +31,13 @@
     "watch": "watch \"yarn on:change\" --ignoreDirectoryPattern \"/dist/\""
   },
   "dependencies": {
+    "@algolia/autocomplete-shared": "1.0.0-alpha.34"
+  },
+  "devDependencies": {
     "@algolia/client-search": "4.5.1",
```

Comment:
is this one only used in tests? otherwise it should be also in peerDependencies; even if it's a transitive dependency (I think).

Reply:
It's to use the `Hit` type: https://github.com/algolia/autocomplete.js/blob/25316b82e0fbe2d3091e41d572c338e825e87a15/packages/autocomplete-preset-algolia/src/highlight/parseAlgoliaHitHighlight.ts#L2

So yeah it should be in peer deps I believe.

Reply:
Added in https://github.com/algolia/autocomplete.js/pull/359/commits/d6ec55ef22866b0a13dae797cfa37e2a1ccdc0b6.